chirp/drivers/kguv980p.py

```diff
@@ -1249,7 +1287,8 @@ def _get_settings(self):
         vfo_grp.append(vfob_grp)
         vfoa_grp.append(vfo150_grp)
         vfoa_grp.append(vfo450_grp)
-        if self.MODEL == "KG-UV980P":
+        if (self.MODEL == "KG-UV980P" or
+           self.MODEL == "KG-UV950P"):
```

Comment:
This is the sort of thing where I'm willing to go along with the use of the model thing, because it's in the middle of a larger block of stuff that's not easy to swap out. But for the str encode/decode you've already got those as separate methods, so you should use the class hierarchy to do the thing for you.